src/plugins/vis_default_editor/public/components/agg_group.tsx

```diff
@@ -70,7 +70,7 @@ function DefaultEditorAggGroup({
   setValidity,
   timeRange,
 }: DefaultEditorAggGroupProps) {
-  const groupNameLabel = (search.aggs.aggGroupNamesMap() as any)[groupName];
+  const groupNameLabel = AggGroupLabels[groupName];
```

Comment:
I simplified the `aggGroupNamesMap` so it didn't require casting or calling a function